docs/6.x/pack.md

```diff
@@ -805,6 +811,33 @@ correct image references.
     There is a sample application available on [GitHub](https://github.com/gravitational/quickstart/tree/master/mattermost)
     that demonstrates this workflow.
 
+### Customizing Helm values
+
+It is possible to customize values of your Helm charts when installing or
+upgrading the application. To provide custom Helm values at install time,
+pass them via `--values` and `--set` flags to `gravity install` command:
+
+```bash
+unpacked-image$ ./gravity install --values=custom-values.yaml --set=nginx.image=1.9.1 --set=postgres.registry=internal.registry.io
+```
+
+The provided values are merged into a single values file that is mounted
+into install and post-install hooks under `/var/lib/gravity/helm/values.yaml`
```

Comment:
Nit: Something to think about, is since this runs within hooks, whether it really makes sense long term to keep `/var/lib/gravity/...` here or whether a flatter structure might make sense at some point.

Reply:
What do you mean by flatter structure?

Reply:
I guess this means that instead of mounting the resources as `/var/lib/gravity/resources/..` they're instead available as `resources/...` - ditto for `helm`.

Reply:
You mean like instead of mounting everything under `/var/lib/gravity` just have `/resources` and `/helm` under root? We could do that at some point, shouldn't be too hard via symlinking to preserve backwards compatibility.

Reply:
yea, some sort of structure that makes sense with some thought. `/config/helm` /config/resources`, etc.